targets/freertos.armv7m/tivadriverlib/sources

```diff
@@ -0,0 +1,15 @@
+include $(OPENMRNPATH)/etc/tivaware.mk
+DEPS  += TIVAWAREPATH
+VPATH := $(TIVAWAREPATH)/driverlib
+
+CFLAGS += -Dcodered
+
+FULLPATHCSRCS = $(wildcard $(VPATH)/*.c)
```

Comment:
It seems you could get away without duplicating these lines if you remove the sources file, and move the above few lines into Makefile of the current directory. 
It might be possible that a small change is needed to etc/lib.mk along the lines of 

ifndef VPATH
VPATH = $(SRCDIR)
endif

and you may have to play with the order of the variable set vs include.